en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md

```diff
@@ -1,80 +1,17 @@
-                "Name": "apim:api_delete",
-                "Roles": "admin,Internal/creator,custom_role"
-            },
-            {
-                "Name": "apim:subscribe",
-                "Roles": "admin,Internal/subscriber,custom_role"
-            },
-            {
-                "Name": "apim:app_owner_change",
-                "Roles": "admin,custom_role"
-            },
-                "Name": "apim:app_import_export",
-                "Roles": "admin,custom_role"
-            },
-            {
-                "Name": "apim:api_import_export",
-                "Roles": "admin,custom_role"
-            },
-            },
-                "Name": "apim:app_manage",
-                "Roles": "admin,Internal/subscriber,custom_role"
-            },
-            {
-                "Name": "apim:api_product_import_export",
-                "Roles": "admin,custom_role"
-            },
-            ``` 
-
-    2.  Using API-M management console.
-        
-        1. Log in to the Admin portal (`https://<APIM_Host>:<APIM_Port>/admin`).
-
-        2. Navigate to **Settings > Scope Mapping** in Admin portal.
-
-        3.  Add `custom_role` to the `Roles` field under the corresponding scope `Name` fields as shown below.
-
-            [![Admin portal scope mapping]({{base_path}}/assets/img/learn/api-controller/admin-portal-scope-mapping.png)]({{base_path}}/assets/img/learn/api-controller/admin-portal-scope-mapping.png)
-
-            The above screenshot only shows adding the `custom_role` to the scopes `apim:api_publish`, `api_view` and `apim:subscribe`. But similarly, you should add `custom_role` to the scopes `apim:app_owner_change`, `apim:app_import_export`, `apim:api_import_export`, `apim:app_manage` and `apim:api_product_import_export` as well.
+## Minimal Permissions and Scopes Required to Perform API Controller Operations
 
-3.  Restart the server or wait for 15 mins until the registry cache expires.
+Further, you can create your own custom user with a custom role to perform specific set of API Controller operations. Refer the below table to learn about the required permissions and scopes that are needed for each of the API Controller operation.
 
-## Minimal Permissions and Scopes Required to Perform API Controller Operations
+!!! info
+    **Steps to create a custom user with a custom role for your need** 
 
-As explained in the above section, you can create any user with a custom role to perform operations using the API Controller. But if you want that user to perform some of the API Controller operations (not all) you can refer the below table and assign the permissions/scopes accordingly for that role.
+    - As shown in [Create user roles]({{base_path}}/administer/managing-users-and-roles/managing-user-roles/#create-user-roles) section, you can create your own custom user role by assigning permissions and scopes that are required to perform a particular set of API Controller operations by reffering the table below.
```

Comment:
At the end of the second line, shall we make it "referring"?

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3097/commits/74cdaa1323d12a99abd038f41a690914b8d50ab5

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3098/commits/edd8ae4c020155b3c9d8e4465a923a031e8f9472 for 3.2.0